### PR TITLE
Update templates to use HTML entities for μ

### PR DESCRIPTION
**Pull request by juananpe**

### CODE REVIEW

File: 22_gui/design_dosages.html [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <html>
   <head>
     <title>Templated Plate Editor with Dosages</title>
-    <script src="./static/alpine.js" defer></script>
-    <link rel="stylesheet" href="./static/page.css">
+    <script src="../static/alpine.js" defer></script>
```
Comment (gvwilson) [inline 22_gui/design_dosages.html:5]: why `../static` ? it works for me referencing `./static`

File: 22_gui/design_dosages.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h1>Templated Plate Editor with Dosages</h1>
       })"
       x-data="{
         treatments: ['CQZ-1718', 'CQA-986', 'CQA-940', '', ],
-        dosages: ['5μL', '10μL', '20μL', '50μL', ],
+        dosages: ['5&mu;L', '10&mu;L', '20&mu;L', '50&mu;L', ],
```
Comment (gvwilson) [inline 22_gui/design_dosages.html:26]: do the `μ` characters not render properly for you? (After some complaints about anglo-centric computing, I'm trying to use açcénted characters and non-Latin characters as-is where I can.)

File: 22_gui/design_dosages.html (hunk 1 above)
Comment (juananpe) [reply 22_gui/design_dosages.html:5]: If I run `make serve` the `./static` links don't work:

image

It is because in https://github.com/gvwilson/web-tutorial/tree/main/docs/22_gui there is no link to `../static`. 
I wonder if we should just use `../static` instead of adding a symlink.

File: 22_gui/design_dosages.html (hunk 2 above)
Comment (juananpe) [reply 22_gui/design_dosages.html:26]: It is not working:
image

BUT, I've just figured out that there is a simpler fix than using HTML encoded entities... just specify that we are using utf-8 using `<meta charset="UTF-8">`

Comment (juananpe) [reply 22_gui/design_dosages.html]: Pushed the meta charset fix.

File: 22_gui/design_dosages.html (hunk 1 above)
Comment (gvwilson) [reply 22_gui/design_dosages.html:6]: ah - I didn't run `make serve` to load the GUI, I just opened the page directly in the browser without a server. The problem with `../static` and the like is that Flask mostly doesn't like loading things that are above the server in the filesystem (to prevent people escaping the sandbox). How about we leave `./static` in this example for now and think about a more general solution to managing `static`, `data`, and `migrations`?

File: 22_gui/design_dosages.html (hunk 2 above)
Comment (gvwilson) [reply 22_gui/design_dosages.html]: oh nice - we should add that to the lesson ("three ways to represent non-7-bit characters and why this is the right one")

File: 22_gui/design_dosages.html (hunk 1 above)
Comment (juananpe) [reply 22_gui/design_dosages.html:6]: I think that the problem is deeper than that. This is what I think:
* `make serve` is not using Flask (it is using Python's internal http.server module)
*  The problem is that the static (and data and migrations) link was not in docs/22_gui (as it should be)
*  Following the thread, I've noticed that in the target render of the Makefile, which in turn calls `mccole render`, the `--symlinks` parameter is not passed, thus the links to static/, data/, and migrations/ were not created in docs/22_gui.
* But passing that `--symlinks` option to the render target yields this error:
```
(web-tutorial)  /opt/web-tutorial | static > make render
mccole render  --symlinks
Traceback (most recent call last):
  File "/opt/web-tutorial/.venv/bin/mccole", line 8, in <module>
    sys.exit(clui.main())
             ^^^^^^^^^^^
  File "/opt/web-tutorial/.venv/lib/python3.12/site-packages/mccole/clui.py", line 42, in main
    render(opt)
  File "/opt/web-tutorial/.venv/lib/python3.12/site-packages/mccole/render.py", line 28, in render
    copy_symlink(opt.out, filepath)
TypeError: copy_symlink() missing 1 required positional argument: 'source_path'
make: *** [render] Error 1
```
* It is due to the way the `copy_symlink` function is being called in `mccole/render.py`
In the `render` function, we are calling `copy_symlink(opt.out, filepath)`, but `copy_symlink` expects three arguments: `output_dir`, `renames`, and `source_path`.
* So it should be like this:
```python
if opt.symlinks:
    for filepath in find_symlinks(opt, skips):
        copy_symlink(opt.out, config["renames"], filepath)
```
Now everything works as expected. I've seen that the `--symlinks` parameter recreates all the links in every folder (not just in the folders that are missing the links...) Just a minor issue.

File: Makefile [hunk 3]
```diff
@@ -26,7 +26,7 @@ lint:
 
 ## render: convert to HTML
 render:
-	mccole render ${CSS}
+	mccole render ${CSS} # use --symlinks to regenerate all symlinks if needed.
```
Comment (gvwilson) [inline Makefile:29]: hm - people shouldn't have to edit the Makefile before running it - can/should we provide a second target `render-symlinks` or something like that? or re-think the whole symlink approach? it's proving to be a pain in the ass.

File: docs/01_intro/data [hunk 4]
```diff
@@ -0,0 +1 @@
+../data
```
Comment (gvwilson) [inline docs/01_intro/data:1]: yeah - so all the `./docs` directories now have these symlinks…

File: 22_gui/design_dosages.html [hunk 5]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="UTF-8">
```
Comment (gvwilson) [inline 22_gui/design_dosages.html:4]: so this is now bundling two changes:
1. character set to handle micro
2. copying symlinks
should those be separate PRs?

Comment (juananpe) [reply 22_gui/design_dosages.html:4]: Indeed. I've just pushed a mini-PR that adds the meta charset.
I will close this PR w/o merging while we discuss #84
